plugins/hls-explicit-imports-plugin/src/Ide/Plugin/ExplicitImports.hs

```diff
@@ -61,12 +61,14 @@ importCommandId = "ImportLensCommand"
 data Log
   = LogShake Shake.Log
   | LogWAEResponseError ResponseError
-  deriving Show
+  | forall a. (Pretty a) => LogResolve a
+
 
 instance Pretty Log where
   pretty = \case
     LogShake logMsg -> pretty logMsg
     LogWAEResponseError rspErr -> "RequestWorkspaceApplyEdit Failed with " <+> viaShow rspErr
+    LogResolve msg -> "ResolveHelpers:" <+> pretty msg
```

Comment:
in situations like this we usually don't add a herald at the front, like with `LogShake`. Otherwise it gets very busy